### ISSUE TITLE
Hello world does not work.

### ISSUE THREAD

**Issue by skyrocknroll** [census-instrumentation/opencensus-python#104]
``` Traceback (most recent call last):
  File "main.py", line 19, in <module>
    from opencensus.trace.tracer import Tracer
ImportError: cannot import name 'Tracer' 
```
More Details
```
Python 3.5.2 (default, Nov 23 2017, 16:37:01) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from opencensus.trace import execution_context
>>> from opencensus.trace.tracer import Tracer
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'Tracer'
```

**Comment by liyanhui1228** [comment]
Thanks for filing the issue. The previously named `RequestTracer` was renamed and restructured to `Tracer` recently and I haven't push the new package to Pypi yet, will do a new release shortly which will resolve this issue.

**Comment by liyanhui1228** [comment]
Resolved via #115.
